server/routes/import_wizard/html.py

```diff
@@ -30,3 +31,58 @@ def main():
 @bp.route('/')
 def main_new():
   return render_template('/import_wizard2.html')
+
+
+# This is only used in CSV + SQL backend, converting raw CSV data into resolved
+# CSV.
+@bp.route('/simple/convert', methods=['POST'])
+def convert():
+  RAW_DATA_PATH = os.environ.get("RAW_DATA_PATH")
```

Comment:
I'm wondering if dath path can be an optional parameter? It can help with testing.

Reply:
That's a good idea.